modules/postgres/examples_test.go

```diff
@@ -28,8 +34,8 @@ func ExampleRunContainer() {
 		postgres.WithUsername(dbUser),
 		postgres.WithPassword(dbPassword),
 		testcontainers.WithWaitStrategy(
-			wait.ForLog("database system is ready to accept connections").
-				WithOccurrence(2).
+			wait.ForSQL(nat.Port("5432/tcp"), "postgres", dbURL).
+				WithQuery("SELECT datname FROM pg_database").
```

Comment:
Let's add this for been more explicit about this SQL query:
```suggestion
				// examine the pg_database system catalog for the set of existing databases
				WithQuery("SELECT datname FROM pg_database").
```